catalog/spatial/geocoding/spatial-geocoding-create/src/main/java/org/codice/ddf/spatial/geocoding/create/GeoNamesCreator.java

```diff
@@ -43,12 +38,16 @@ public GeoEntry createGeoEntry(final String line, String entryResource) {
     // Passing a negative value to preserve empty fields.
     final String[] fields = line.split("\\t", -1);
 
+    Optional<CountryCode> optionalCountryCode =
+        isoStandard
+            .getStandardEntries()
+            .stream()
+            .filter(c -> c.getAsFormat("alpha2").equals(fields[8]))
+            .findFirst();
+
     String countryCode = fields[8];
-    Set<CountryCode> countryCodes =
-        converter.fromAlpha2(
-            fields[8].trim(), isoStandard.getStandard(), isoStandard.getStandard());
-    if (CollectionUtils.isNotEmpty(countryCodes)) {
-      countryCode = countryCodes.iterator().next().getAsFormat("alpha3");
+    if (optionalCountryCode.isPresent()) {
```

Comment:
✏️  We could do this as such if we want to get rid of the isPresent check
```java
String countryCode = optionalCountryCode
    .map(cc -> cc.getAsFormat("alpha3")
    .orElse(fields[8]);
```
(in fact we could even skip storing the optionalCountryCode var as well if you wish)

Reply:
That does look cleaner. Thanks!